site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -13,6 +13,8 @@ publishDate: 2023-02-28
 
 When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be **horizontally scaled**, making the Agones control plane **highly available** and more **resiliant to disruption**.
 
+`SplitControllerAndExtensions` also permits multiple `agones-controller` pods and leader eleciton usage. Having multiple `agones-controller` minimizes downtime of the service from voluntary pod disruptions such as pod deletes, deployment updates, and pod restarts.
```

Comment:
Since this isn't going to be true until the next release, this should either be release guarded (see https://agones.dev/site/docs/contribute/#within-a-page) or just held and merged _after_ the release gets cut.

Reply:
Added the release guard